addons-l10n/en/hovercards.json

```diff
@@ -0,0 +1,28 @@
+{
+  "hovercards/post-count": "{count} posts",
+  "hovercards/loading": "Loading...",
+  "hovercards/error-request-failed": "Request failed. {error}",
+  "hovercards/error-scratchdb": "Error encountered on ScratchDB. {error}",
+  "hovercards/error-no-project": "The project couldn't be found or is private.",
+  "hovercards/error-no-studio": "The studio couldn't be found.",
+  "hovercards/error-no-user": "The user couldn't be found.",
+  "hovercards/error-no-user-scratchdb": "The user couldn't be found in ScratchDB. The user may be too new to be indexed.",
+  "hovercards/error-no-user-statistics-scratchdb": "No statistics available. The user is too new to be fully indexed.",
+  "hovercards/error-no-forum-topic": "The topic couldn't be found.",
+  "hovercards/error-deleted-forum-topic-scratchdb": "The topic has been deleted. (at least in ScratchDB)",
+  "hovercards/error-no-forum-post": "The post couldn't be found.",
+  "hovercards/error-deleted-forum-post-scratchdb": "The post has been deleted. (at least in ScratchDB)",
+  "hovercards/info-views": "{count, plural, 0 {No views} one {1 view} other {# views}}",
```

Comment:
Transifex might have a problem for special cased `0`, if possible just use one-other